OpenRA.Mods.Common/Widgets/SupportPowersWidget.cs

```diff
@@ -125,18 +129,59 @@ public class SupportPowerIcon
 			public HotkeyReference Hotkey;
 		}
 
+		bool IsPowersModified()
```

Comment:
This method should ideally be able to go away, with the only change in this file being a `powersModified` check in `Tick` and then registering/unregistering the notification.